tests/cypress/e2e/for-teacher_page/customize_class_page/remove_adventure_button.cy.js

```diff
@@ -0,0 +1,63 @@
+import {loginForTeacher} from '../../tools/login/login.js'
+import { createClass } from "../../tools/classes/class";
+
+describe('Back to class button', () => {
+  it('passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    createClass();
+
+    cy.wait(500);
+
+    cy.get(".view_class").first().click(); // Press on view class button
+    cy.get('#customize-class-button').click(); // Press customize class button
+
+    cy.get('#sortadventures')
+      .children()
+      .should('not.be.visible');
+
+
+    // Click on level 2
+    cy.get("#adventures")
+      .select('2')
+      .should('have.value', '2');
+
+    cy.wait(500);
+    
+    // The available adventures dropdown should only include the default option
+    cy.get("#available")
+      .children()
+      .should('have.length', 1)
+    
+    // store the name of the adventure we're going to delete
+    cy.get('#level-2 div:first')
+      .invoke('attr', 'adventure')
+      .as('adventure');
+    
+    cy.get("@adventure").then(adventure => {
```

Comment:
From reading the docs, I don't think this `as() + get()` is necessary. You can probably do:

```ts
cy.get(...)
   .invoke(...)
   .then(adventure => {
    });
```

Reply:
Yes, good idea!